app/src/main/java/de/danoeh/antennapod/adapter/ChaptersListAdapter.java

```diff
@@ -109,7 +109,7 @@ public View getView(final int position, View convertView, ViewGroup parent) {
                     if (link.length != 0) {
```

Comment:
Up in this block you'll see that action could only be either up or down, so a check after the else is unnecessary.